pkgs/applications/misc/k2pdfopt/default.nix

```diff
@@ -28,6 +28,7 @@ stdenv.mkDerivation rec {
   let
     mupdf_modded = mupdf.overrideAttrs (attrs: {
       name = "mupdf-1.10a";
```

Comment:
```suggestion
      pname = "mupdf";
```

Reply:
Doesn't work, because the `name` attribute is already determined at this point